src/ProductConstructionService/Readme.md

```diff
@@ -44,7 +44,12 @@ Run the `provision.ps1` script by giving it the name of the subscription you wan
 
 This will create all of the necessary Azure resources.
 
-Once the resources are created, go to the newly created User Assigned Managed Identity. Copy the Client ID, and paste it in the correct appconfig.json, under `ManagedIdentityClientId``
+We're using a Managed Identity to authenticate PCS to BAR. You'll need to run the following SQL queries to enable this (we can't run SQL from bicep):
```

Comment:
How about Maestro? Should we put the same in the `DevGuide.md`?

Reply:
I don't think we really have a thing like this with Maestro. We could maybe start writing it down. IMO, `DevGuide.md` is more for running maestro locally, than it is for creating it in Azure